core/deployment/src/main/java/io/quarkus/deployment/pkg/PackageConfig.java

```diff
@@ -302,24 +312,19 @@ public String getRunnerSuffix() {
     }
 
     @ConfigGroup
-    public static class VineFlowerConfig {
+    public static class DecompilerConfig {
         /**
          * An advanced option that will decompile generated and transformed bytecode into the 'decompiled' directory.
          * This is only taken into account when fast-jar is used.
          */
-        @ConfigItem(defaultValue = "false")
-        public boolean enabled;
-
-        /**
-         * The version of Vineflower to use
-         */
-        @ConfigItem(defaultValue = "1.9.3")
-        public String version;
+        @ConfigDocDefault("false")
```

Comment:
Adding `@ConfigItem` should fix https://github.com/quarkusio/quarkus/pull/37445#issuecomment-1864271282

Reply:
Thanks! Let me try it out

Reply:
Thanks! Let me try it out